File: example_2.py
import requests

def send_message_to_aelius(user_input):
    url = 'https://aelius.live/api/v1/chat'
    headers = {'Content-Type': 'application/json'}
    data = {'user_input': user_input}
    
    try:
        response = requests.post(url, headers=headers, json=data)
        return response.json()['response']
    except Exception as e:
        print(f"Error fetching AI response: {e}")
        return None

def main():
    print("Welcome to the Aelius Command-Line Chatbot!")
    while True:
        user_input = input("You: ")
        if user_input.lower() in ['exit', 'quit']:
            print("Exiting chatbot. Goodbye!")
            break
        response = send_message_to_aelius(user_input)
        if response:
            print(f"Assistant: {response}")

main()

File: example_2.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
  terminal: true
});

async function sendMessage(userInput) {
    const fetch = (await import('node-fetch')).default;
    const url = 'https://aelius.live/api/v1/chat';
    const headers = {'Content-Type': 'application/json'};
    const data = {user_input: userInput};
    
    try {
        const response = await fetch(url, {
            method: 'POST',
            headers: headers,
            body: JSON.stringify(data)
        });
        const responseData = await response.json();
        displayMessage('Assistant', responseData.response);
    } catch (e) {
        console.error(`Error fetching AI response: ${e}`);
    }
}

function displayMessage(who, message) {
    console.log(`${who}: ${message}`);
}

// Example usage:
function promptUser() {
    rl.question('Type a message: ', (userInputValue) => {
        if (userInputValue.trim() === 'exit') {
            rl.close();
            return;
        }
        sendMessage(userInputValue);
        promptUser();
    });
}

promptUser();
